### ISSUE TITLE
Files missing

### ISSUE THREAD

**Issue by embeh** [sriveravi/FiducialDetector_Public#1]
Maybe this is intentional, but the example1.m file requires a set of files which are missing from your repo:

load( 'trainTestImageMarkingAR.mat', 'trainImages','trainMarkings', 'testImages' );

Thanks for sharing!

**Comment by sriveravi** [comment]
Good catch.  I left out because I did not have ownership of that data to distribute.  However, there is a description of the format of training data you would need to run the algorithm (See example1.m).  

% trainImages:  height x width x N training images
% trainMarkings:  numLandmarks x N complex landmarks  ( x + yi)
% testImages height x width x Ntest images

**Comment by embeh** [comment]
Is the description available?
